### PR TITLE
Feature/smtp creds

### PR DESCRIPTION
**Pull request by berkes**
This adds config vars to add SMTP login credentials: user/password.

For SMTP-servers (all of them?) that require login.

### CODE REVIEW

**Comment by berkes** [comment]
I did not write tests for this, as I

* Could not find any other tests for specifically the vars being parsed and used in the template.
* Could not get the pip stuff configured without breaking my entire DE.

File: defaults/main.yml [hunk 1]
```diff
@@ -47,6 +47,8 @@ gitea_mailer_skip_verify: false
 gitea_mailer_tls_enabled: true
 gitea_mailer_host: localhost:25
 gitea_mailer_from: noreply@your.domain
+gitea_mailer_user: ""
+gitea_secret_password: ""
```
Comment (DO1JLR) [inline defaults/main.yml]: These variable probably should be renamed to ``gitea_mailer_password``

**Comment by berkes** [comment]
Good catch. I'll follow this up with a force push. Thanks Wed Jun 12 13:36:37 GMT+02:00 2019 thomas-maurice/ansible-role-gitea <reply@reply.github.com>: @DO1JLR requested changes on this pull request. In defaults/main.yml: > @@ -47,6 +47,8 @@ gitea_mailer_skip_verify: false gitea_mailer_tls_enabled: true gitea_mailer_host: localhost:25 gitea_mailer_from: noreply@your.domain +gitea_mailer_user: "" +gitea_secret_password: "" These variable probably should be renamed to gitea_mailer_password — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by berkes** [comment]
The issue is fixed and travis now passes that part.

Travis still fails on running the role on debian-stretch. I assume this is unrelated to this change, though.

**Comment by thomas-maurice** [comment]
@berkes thanks for doing that and thanks @DO1JLR for the review!

Looks good to me, and yes I am aware that the testing is shit right now :/ I have to find a way of making it work properly with molecule and travis

Either way merging, thanks :)

**Pull request merged**